### PR TITLE
Frontend cleanup

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
### Context

* Ensure that some content pages have show their title in the browser window
* Use `l` heading size for pages which are forms
* Remove CSS that is no longer used
* Ensure we are always using `app-*` for app specific classes (have ignored map styles as those are disappearing soon)
* Organise stylesheets and list imports alphabetically (leaving those that need further refactoring or that will disappear soon)

### Changes proposed in this pull request

Have broken out changes into individual commits, and where necessary, added a little more context to the commit message.

### Guidance to review

* Have updated `govuk_link_to` to use same code as used on Apply; this helper was causing issues previously if provided with a block.

### Trello card

https://trello.com/c/Za2awSWs/3050-find-clean-up

### Checklist

- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: app/views/courses/_contact_details.html.erb [hunk 1]
```diff
@@ -2,30 +2,30 @@
   <h2 class="govuk-heading-l" id="section-contact">Contact details</h2>
 
   <div class="course-basicinfo">
-    <dl class="govuk-list--description">
+    <dl class="app-list--description">
```
Comment (frankieroberto) [inline app/views/courses/_contact_details.html.erb:5]: Is this a app-specific component, or a modification of a `govuk-frontend` one? I don't think `govuk-list` is meant to be used with definition lists?

File: app/views/pages/cycle_ending_soon.html.erb [hunk 2]
```diff
@@ -3,21 +3,20 @@
...
-        You’ll be able to submit an application from 13 October 2020.
-      </p>
+      <p class="govuk-body">Come back from 6 October 2020 to find courses.</p>
+      <p class="govuk-body">You’ll be able to submit an application from 13 October 2020.</p>
+
       <h2 class="govuk-heading-m">If you’ve already applied</h2>
       <p class="govuk-body">
-        To help candidates who've already applied get placed on an available course,
-        <%= link_to "this website is still showing some courses (these may not be available for the 2021 to 2022 academic year)", start_location_path, class: "govuk-link" %>
+        To help candidates who’ve already applied get placed on an available course,
```
Comment (frankieroberto) [inline app/views/pages/cycle_ending_soon.html.erb:18]: 👀 👍

File: app/views/pages/privacy.html.erb [hunk 3]
```diff
@@ -39,17 +37,16 @@
       </p>
 
       <h2 class="govuk-heading-l">The right to lodge a complaint</h2>
-
-      <p class="govuk-body">You have the right to raise any concerns with the Information Commissioner’s Office (ICO) via their website at <a href="https://ico.org.uk/concerns/" class="govuk-link">https://ico.org.uk/concerns/</a>.
+      <p class="govuk-body">
+        You have the right to raise any concerns with the Information Commissioner’s Office (ICO) via their website at
+        <%= govuk_link_to('https://ico.org.uk/concerns/', 'https://ico.org.uk/concerns/') %>.
```
Comment (frankieroberto) [inline app/views/pages/privacy.html.erb:42]: This URL seems to have been redirected...

```suggestion
        <%= govuk_link_to('https://ico.org.uk/make-a-complaint', 'https://ico.org.uk/make-a-complaint') %>.
```

File: app/views/pages/terms.html.erb [hunk 4]
```diff
@@ -1,34 +1,26 @@
...
-        </p>
+      <h2 class="govuk-heading-l">Using our website</h2>
+      <p class="govuk-body">The ‘Find postgraduate teacher training’ service is maintained for your personal use and viewing. Access and use by you of this site constitutes your acceptance of these terms and conditions. This takes effect from the date on which you first use this website.</p>
 
-        <h2 class="govuk-heading-l">Virus protection</h2>
-        <p class="govuk-body">
-          We make every effort to check and test material at all stages of production. It is always wise for you to run an anti-virus program on all material downloaded from the Internet. We cannot accept any responsibility for any loss, disruption or damage to your data or your computer system which may occur whilst using material derived from this website.
-        </p>
+      <h2 class="govuk-heading-l">Hyperlinking to Find postgraduate teacher training </h2>
```
Comment (frankieroberto) [inline app/views/pages/terms.html.erb:11]: 😆 

```suggestion
      <h2 class="govuk-heading-l">Linking to Find postgraduate teacher training </h2>
```

File: app/views/results/_non_university.html.erb [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <% nearest_address = @results_view.nearest_address(course) %>
 <% nearest_location_name = @results_view.nearest_location_name(course) %>
 
-<dt class="govuk-list--description__label">
+<dt class="app-list--description__label">
```
Comment (frankieroberto) [inline app/views/results/_non_university.html.erb:5]: As above, not sure this is intended as a modifier of `govuk-list` or a custom app component.

**Review by frankieroberto — COMMENTED** [review]
Nice work.

My only question is whether `app-list` is right, given that this doesn't _seem_ to be a modification of `govuk-list`, and if it's a custom app-specific component, maybe we should give it a slightly different name. Alternatively, perhaps it's closer to a modification of `govuk-summary-list`?

File: app/views/courses/_contact_details.html.erb (hunk 1 above)
Comment (paulrobertlloyd) [reply app/views/courses/_contact_details.html.erb]: Good point. Updated this from `govuk-list--description` to `app-description-list`

File: app/views/pages/privacy.html.erb (hunk 3 above)
Comment (paulrobertlloyd) [reply app/views/pages/privacy.html.erb]: Nice spot! Updated.

File: app/views/pages/terms.html.erb (hunk 4 above)
Comment (paulrobertlloyd) [reply app/views/pages/terms.html.erb]: Eww… updated.

**Review by TobyRet — APPROVED** [review]

**Pull request merged**